source/usr/lib/mcvirt/auth/user.py

```diff
@@ -18,14 +18,21 @@
 # along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
 
 from mcvirt.auth.user_base import UserBase
-from mcvirt.exceptions import OldPasswordIncorrect
+from mcvirt.auth.permissions import PERMISSIONS
+
+import Pyro4
 
 
 class User(UserBase):
     """Provides an interaction with the local user backend"""
 
-    def change_password(self, old_password, new_password):
+    @Pyro4.expose()
+    def change_password(self, new_password):
```

Comment:
As suggested previously, it's probably better to call this 'set_password' since it doesn't require the previous password and is mainly just a wrapper around _set_password